src/CarbonAware.WebApi/src/Filters/HttpResponseExceptionFilter.cs

```diff
@@ -18,6 +18,7 @@ public class HttpResponseExceptionFilter : IExceptionFilter
     {
         { "ArgumentException", (int)HttpStatusCode.BadRequest },
         { "NotImplementedException", (int)HttpStatusCode.NotImplemented },
+        { "InvalidOperationException", (int)HttpStatusCode.BadRequest },
```

Comment:
Trying to understand why this was changed to InvalidOperation

Reply:
When the CA Library handlers run into configuration argument issues (like the location being null), they end up throwing InvalidOperationException and returning an Internal Server Error. I added this here to return as a BadRequest instead because this matched with the integration tests previously written for the Aggregators (they are expecting BadRequest).

We can definitely change this though if it makes more sense: I can remove the InvalidOperationException snippet, let it throw Internal Error instead, and modify the integration tests?

Reply:
No, I think any validation error should be a BadRequest, so what you did makes sense. I am wondering why the handlers return InvalidOperationException, but that's not in scope for your PR and needs to be investigated separately.

Reply:
That makes sense! I can open a spike/story to investigate this